custom_components/tahoma/__init__.py

```diff
@@ -174,7 +176,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         if device.widget == HOMEKIT_STACK:
             print_homekit_setup_code(device)
 
-    for platform in platforms:
+    supported_platforms = set(platforms.keys())
+
+    # Sensor and Binary Sensor will be added dynamic, based on the device states
```

Comment:
dynamic and dynamically have the same meaning?
```suggestion
    # Sensor and Binary Sensor will be added dynamically, based on the device states
```